src/client/index.tsx

```diff
@@ -1,18 +1,48 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
+interface ConsultantParameter {
+    label: string;
+    value: string;
+}
 
-const name = "John Doe";
+const consultantLines: ConsultantParameter[] = [
+    {
+        label: "Name",
+        value: "John Doe",
+    },
+];
+
+/* consultantLines.map((element: ConsultantParameter) => {
+    return (
+        <div>
+            {element}
+        </div>
+    );
+}) */
+
+/* const consultantRow: JSX.Element = (
+    <div className={"tableRow"}>
+        <div className={"tableCell"}>{consultantLines[0].label}</div>
+        <div className={"tableCell"}>{consultantLines[0].value}</div>
+    </div>
+); */
 
 const consultantElement: JSX.Element = (
     <div className={"box"}>
         <div className={"tableRow"}>
             <div className={"tableCell"}>{"Consultant"}</div>
         </div>
-        <div className={"tableRow"}>
-            <div className={"tableCell"}>{"Name"}</div>
-            <div className={"tableCell"}>{name}</div>
-        </div>
+        {consultantLines.map((element: ConsultantParameter) => {
+            return (
+                <div key={"element"}>
+                    <div className={"tableRow"}>
```

Comment:
`key` can be in here, together with `className`. And it should be different for each element, but currently it'll have same "element" value.

Reply:
It's genius!